test/integration/targets/junos_rpc/tests/netconf/rpc.yaml

```diff
@@ -10,7 +10,7 @@
 - assert:
     that:
       - "result.changed == false"
-      - "'<name>\nem0\n</name>' in result['xml']"
+      - "'<interface-information' in result['xml']"
```

Comment:
This isn't correct, we need to drop the single quote:

  https://logs.zuul.ansible.com/01/56601/0ddf48a36c67950e70abe358a8bb43fb4a9cf8d9/third-party-check/ansible-test-network-integration-junos-python37/d38da69/controller/ara-report/result/9ce267ed-c1b3-4ff8-9d61-69531b7807d7/